app/src/cli/commands/clone.ts

```diff
@@ -0,0 +1,69 @@
+import * as QueryString from 'querystring'
+import { URL } from 'url'
+
+import * as yargs from 'yargs'
```

Comment:
Could you update that for me? I’m on `npm@5`, so it rewrites the whole `npm-shrinkwrap.json`.